lib/config/urls.dart

```diff
@@ -11,6 +11,9 @@ abstract class Url {
   /// Dev url.
   static String get devUrl => 'https://localhost:9999/food/hs/ExchangeSotr';
 
+  /// Url for getting IP address.
+  static const String ipUrl = 'https://api.ipify.org/';
+
```

Comment:
Remove variable here and assign it directly in `api_intercat_example_scope`, please.